data/schema.yaml

```diff
@@ -106,42 +105,33 @@ components:
           type: string
           format: date-time
           example: 2017-12-21T17:32:28Z
-        MetaDescription:
+        metaDescription:
           type: boolean
           example: null
```

Comment:
A boolean with the example of `null` is wrong, should be true or false. There are a few more example of this, can we change them all please

Reply:
Yep